importme.js

```diff
@@ -0,0 +1,2 @@
+require('./dist/angular-google-maps')
+module.exports = 'uiGmapgoogle-maps'
```

Comment:
why not have this as the index.js?

Reply:
Because I could not oversee if that would impact other build tools. I've just recently picked up Webpack and never really gave grunt much attention

Reply:
I guess you could add it to "main" in the package.json

Reply:
add `importme.js` to the `main` entry?

Reply:
@devotis yeah exacltly